samples/meeting-tabs/python/components/share-to-meeting.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
```

Comment:
Please use the latest Team-JS SDK CDN Link and test the sample.

Latest version
@microsoft/teams-js@2.40.0